website/blog/2023-05-28-databend-weekly-95.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "This Week in Databend #95"
+date: 2023-05-28
+slug: 2023-05-28-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: ariesdevil
+  - name: BohuTANG
+  - name: dantengsky
+  - name: Dousir9
+  - name: everpcpc
+  - name: flaneur2020
+  - name: gitccl
+  - name: jun0315
+  - name: leiysky
+  - name: lichuang
+  - name: Mehrbod2002
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: SkyFan2002
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhengLin-Li
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Native COS Support
+
+Databend now offers native support for COS! Compared to the S3 compatibility provided by COS, our native support has the following advantages:
+
+- By supporting cos's native signature algorithm, we have fixed some signing errors in certain boundary cases.
+- In the future, we will provide append and other functions that are not supported by s3 based on COS.
+- Through COS's native support, we can integrate with Tencent Cloud's RAM, STS and other services to provide authentication capabilities without static keys.
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [PR | feat(storage): Add native COS support](https://github.com/datafuselabs/databend/pull/11559)
+
+### IEJoin
+
+Inequality joins are commonly used in various applications to join relational tables based on inequality conditions. Databend has recently added support for IEJoin.
```

Comment:
Databend has recently introduced support for IEJoin (Inequality Join), which is commonly used in various applications to join relational tables based on inequality conditions.